### PR TITLE
feat: add test cases for services plugin.

### PR DESCRIPTION
**Pull request by mjatin-dev**

### CODE REVIEW

File: admin-ui/app/utils/AppAuthProvider.js [hunk 1]
```diff
@@ -41,8 +41,7 @@ class AppAuthProvider extends Component {
       console.warn('Parameters to process authz code flow are missing.')
       return
     }
-    return `${authzBaseUrl}?response_type=${responseType}
-     &redirect_uri=${redirectUrl}&client_id=${clientId}&scope=${scope}&state=${state}&nonce=${nonce}`
+    return `${authzBaseUrl}?response_type=${responseType}&redirect_uri=${redirectUrl}&client_id=${clientId}&scope=${scope}&state=${state}&nonce=${nonce}`;
```
Comment (syntrydy) [inline admin-ui/app/utils/AppAuthProvider.js:44]: This line should  be formatted, i mean the whole file

File: admin-ui/plugins/services/Components/Configuration/CacheInMemory.test.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
+import React from "react";
```
Comment (syntrydy) [inline admin-ui/plugins/services/Components/Configuration/CacheInMemory.test.js:1]: The single quote is used everywhere not a double quote.

File: admin-ui/plugins/services/Components/Configuration/CacheInMemory.test.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+import { render, screen } from "@testing-library/react";
+import CacheInMemory from "./CacheInMemory";
+import { createStore, combineReducers } from "redux";
+import { Provider } from "react-redux";
+import i18n from "../../../../app/i18n";
```
Comment (syntrydy) [inline admin-ui/plugins/services/Components/Configuration/CacheInMemory.test.js:6]: No need for a semi-colon after each instruction, refer to other pages

File: admin-ui/plugins/services/Components/Configuration/CacheMemcached.js [hunk 4]
```diff
@@ -1,20 +1,20 @@
...
   Col,
   CardBody,
   CustomInput,
-} from '../../../../app/components'
-import GluuLabel from '../../../../app/routes/Apps/Gluu/GluuLabel'
-import { CACHE } from '../../../../app/utils/ApiResources'
-import GluuTooltip from '../../../../app/routes/Apps/Gluu/GluuTooltip'
-import GluuInputRow from '../../../../app/routes/Apps/Gluu/GluuInputRow'
-import { useTranslation } from 'react-i18next'
+} from "../../../../app/components";
```
Comment (syntrydy) [inline admin-ui/plugins/services/Components/Configuration/CacheMemcached.js:9]: same here

File: admin-ui/app/utils/AppAuthProvider.js (hunk 1 above)
Comment (mjatin-dev) [reply admin-ui/app/utils/AppAuthProvider.js:44]: With formatter it gives the same result.

File: admin-ui/plugins/services/Components/Configuration/CacheInMemory.test.js (hunk 3 above)
Comment (mjatin-dev) [reply admin-ui/plugins/services/Components/Configuration/CacheInMemory.test.js:6]: Fixed

File: admin-ui/plugins/services/Components/Configuration/CacheMemcached.js (hunk 4 above)
Comment (mjatin-dev) [reply admin-ui/plugins/services/Components/Configuration/CacheMemcached.js:9]: Fixed

File: admin-ui/plugins/services/Components/Configuration/CacheInMemory.test.js (hunk 2 above)
Comment (mjatin-dev) [reply admin-ui/plugins/services/Components/Configuration/CacheInMemory.test.js:1]: Fixed

**Pull request closed without merging**